### PR TITLE
Add parallax to demo app with authors list

### PR DESCRIPTION
**Pull request by andydotxyz**
https://github.com/fyne-io/fyne/assets/294436/427a02fd-63c3-42d3-a71b-ccd2cc92c9dc

### Checklist:

- [ ] Tests included.
- [x] Lint and formatter run with no errors.
- [x] Tests all pass.

### CODE REVIEW

File: cmd/fyne_demo/tutorials/welcome.go [hunk 1]
```diff
@@ -28,16 +32,122 @@ func welcomeScreen(_ fyne.Window) fyne.CanvasObject {
...
+`)
+
+	for _, line := range strings.Split(lines, "\n") {
+		if len(line) == 0 {
+			continue
+		}
+
+		markdown.WriteString("* ")
+		markdown.WriteString(line)
+		markdown.WriteString("\n")
```
Comment (Jacalz) [inline cmd/fyne_demo/tutorials/welcome.go:149]:
```suggestion
		markdown.WriteByte('\n')
```

File: cmd/fyne_demo/tutorials/welcome.go [hunk 2]
```diff
@@ -28,16 +32,122 @@ func welcomeScreen(_ fyne.Window) fyne.CanvasObject {
...
+
+func (u unpad) MinSize(_ []fyne.CanvasObject) fyne.Size {
+	return fyne.NewSize(100, 100)
+}
+
+func formatAuthors(lines string) string {
+	markdown := &strings.Builder{}
+	markdown.WriteString(`### Authors
+
+`)
```
Comment (Jacalz) [inline cmd/fyne_demo/tutorials/welcome.go:140]: A little more clear to just have `"### Authors\n\n"`?

File: cmd/fyne_demo/data/metadata_bundled.go [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+// Code generated by '$ fyne bundle'. DO NOT EDIT.
+
+package data
+
+import "fyne.io/fyne/v2"
+
+var resourceAuthors = &fyne.StaticResource{
+	StaticName: "AUTHORS",
+	StaticContent: []byte(
+		"Andy Williams <andy@andy.xyz>\nSteve OConnor <steveoc64@gmail.com>\nLuca Corbo <lu.corbo@gmail.com>\nPaul Hovey <paul@paulhovey.org>\nCharles Corbett <nafredy@gmail.com>\nTilo Prütz <tilo@pruetz.net>\nStephen Houston <smhouston88@gmail.com>\nStorm Hess <stormhess@gloryskulls.com>\nStuart Scott <stuart.murray.scott@gmail.com>\nJacob Alzén <jacalz@tutanota.com>\nCharles A. Daniels <charles@cdaniels.net>\nPablo Fuentes <f.pablo1@hotmail.com>\nChangkun Ou <hi@changkun.de>\n\n"),
```
Comment (Jacalz) [inline cmd/fyne_demo/data/metadata_bundled.go:11]: We never use `StaticName` here. Would it not be nicer to just use `go:embed` and store the file data that way (more automated)?

**Review by Jacalz — CHANGES_REQUESTED** [review]
Nice change. A few suggestions inline :)

File: cmd/fyne_demo/data/metadata_bundled.go (hunk 3 above)
Comment (andydotxyz) [reply cmd/fyne_demo/data/metadata_bundled.go:11]: We cannot as they will not allow "../../" in the file path.

Comment (Jacalz) [reply cmd/fyne_demo/data/metadata_bundled.go:11]: Alright 👍

**Review by Jacalz — APPROVED** [review]

**Pull request merged**